src/pages/EduHomework/EduHomework.jsx

```diff
@@ -3,14 +3,15 @@ import './eduhomework.scss';
 import EduHomeworkTable from '../../components/EduHomeworkTable/EduHomeworkTable';
 import HeadingPageComponent from '../../components/HeadingPageComponent/HeadingPageComponent';
 import { Link } from 'react-router-dom';
-
+import EduHomeworkComponent from '../../components/EduHomeworkComponent/EduHomeworkComponent';
 
 export default function EduHomework() {
 
     return (
         <div className='eduHomeworkPage'>
             <HeadingPageComponent title={'Edu Homework'} />
             <Link className='eduHomeworkPage__btn' to='/student-tracker'>&larr; goBack</Link>
+            <EduHomeworkComponent />
```

Comment:
This component shouldn't be included here. EduHomework Component is for the /students/:profile page(which is not created yet).

Reply:
Changes made accordingly.

Reply:
Good, job guys, fantastic.